### PR TITLE
Add taxjar credentials resource

### PR DESCRIPTION
**Pull request by dariashka**

### CODE REVIEW

File: src/resources/taxjar-credentials-resource.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export default function TaxJarCredentialsResource({apiHandler}) {
```
Comment (agualis) [inline src/resources/taxjar-credentials-resource.js:1]: I miss the "autogenerated header" in this file. Was it manually generated?

We should use [rebilly-sdk-generator](https://github.com/Rebilly/rebilly-sdk-generator/blob/f94eb73b9316978e9b92785dda2f198007f3eb70/package.json#L17) instead.

**Review by agualis — CHANGES_REQUESTED** [review]

File: src/resources/taxjar-credentials-resource.js (hunk 1 above)
Comment (dariashka) [reply src/resources/taxjar-credentials-resource.js]: It does not generate the methods for tax-jar, is there something wrong with API definitions?

**Review by ZengJunyong — APPROVED** [review]
Approve it. we can figure why the auto-generated did not work well out in this case later.

**Comment by dariashka** [comment]
We should fix it and make it auto-generated too for the future. I created an [issue](https://github.com/Rebilly/rebilly-sdk-generator/issues/20).

**Pull request merged**